### ISSUE TITLE
Is the HAR model only trained on the artificial data or evaluated as well?

### ISSUE THREAD

**Issue by qingxinxia** [qingxinxia/OpenPackChallenge2025#3]
The training set is (a part of the real Openpack data of Scenario 1 + virtual data), and the test set is (the rest of the real Openpack data of Scenario 1).

We will change the random seed and split portion of the training/set sets during evaluation. So, be sure your algorithm can work well on various settings.
